apps/rush-lib/src/logic/PackageJsonUpdater.ts

```diff
@@ -306,67 +306,98 @@ export class PackageJsonUpdater {
 
     await installManager.ensureLocalPackageManager();
     let selectedVersion: string | undefined;
+    const rushConfigProjects: RushConfigurationProject[] = this._rushConfiguration.projects;
 
     if (initialSpec && initialSpec !== 'latest') {
```

Comment:
We probably want the local package check to happen even if a version specification isn't provided.